packages/database/src/migrations/20210804063133-AddFijiCOVIDDashboardTestsByDay-modifies-data.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+import {
+  insertObject,
+  generateId,
+  findSingleRecord,
+  findSingleRecordBySql,
+  deleteObject,
+} from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const generateConfig = () => ({
+  code: `FJ_Covid_Tests_By_Day`,
+  reportConfig: {
+    fetch: {
+      aggregations: [
+        {
+          type: 'RAW',
+          config: {
+            dataSourceEntityType: 'facility',
+          },
+        },
+      ],
+      dataElements: ['COVIDTest_FJNumTest'],
+    },
+    transform: [
+      {
+        '...': 'drop',
+        value: 'sum',
+        period: 'group',
+        transform: 'aggregate',
+      },
```

Comment:
Nothing major, but I think I have a slight preference to doing aggregations in the `aggregations` part of `fetch`. eg. `SUM_EACH_DAY`.

We've been introducing performance optimisations to that process, where we actually run the aggregation itself at the database level that can substantially speed up fetch speed in some cases. 

I suspect in this case it probably won't make much of a difference (unless there were many responses per day, then it would actually be rather useful) so consider it optional, but just wanted to mention my general preference towards aggregating during the fetch if possible 👍

Reply:
Thanks Rohan, this is fantastic :))